test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,43 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
Recall from tutorial if putting all possible test condition into one Junit test or separating the test conditions into multiple Junit test is better.